packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -52,6 +52,7 @@ const BASIS_POINTS = 10_000n;
  * @typedef { WellKnownSpaces & ChainBootstrapSpace & EconomyBootstrapSpace
  * } EconomyBootstrapPowers
  * @typedef {PromiseSpaceOf<{
+ *   economicCommitteeKit: CommitteeStartResult,
```

Comment:
This is one of a few cases I found where we didn't capture the whole `startInstance` result; in particular, we didn't capture the `adminFacet`, so we won't be able to upgrade. I'm inclined to #6034.